src/pages/core/getting-started/rust.md

```diff
@@ -4,4 +4,100 @@ metaTitle: Core - Getting Started - Rust
 description: Get started with Core using Rust
 ---
 
-_coming soon_
\ No newline at end of file
+## Resources
+
+| Site          | Link                                                                   |
+| ------------- | ---------------------------------------------------------------------- |
+| crates.io     | [https://crates.io/crates/mpl-core](https://crates.io/crates/mpl-core) |
+| rust autodocs | [https://docs.rs/mpl-core/](https://docs.rs/mpl-core/)                 |
+
+## Installation
+You can add the `mpl-core` crate to your Rust project by installing with `cargo`.
+
+```shell
+cargo add mpl-core
+```
+
+Alternatively you can paste the version of `mpl-core` crate you wish to use straight into your Cargo.toml file under `[dependencies]`.
+
+```toml
+mpl-core = "0.6.1"
```

Comment:
If you exclude the version here it should just add the latest, which I think is what we want. Just so we don't have to always update this.

Reply:
sweet, will replace with x.x.x